coffeecard/CoffeeCard.WebApi/Services/EmailService.cs

```diff
@@ -77,16 +74,11 @@ public void SendRegistrationVerificationEmail(User user, string token)
         public void SendVerificationEmailForChangedEmail(User user, string token, string newEmail)
```

Comment:
Consider outcommenting code and throw a `NotImplementedException`

Reply:
Also we need to add a note to rethink the update email flow - should the email be updated before the new one is verified etc..

Reply:
Added a todo and now throws the notImplemented exception as the first ting